rel/i18n/zh/emqx_bridge_mongodb.hocon

```diff
@@ -0,0 +1,57 @@
+emqx_bridge_mongodb {
```

Comment:
the zh files are to be moved to
https://github.com/emqx/emqx-i18n

Reply:
ok, so I just delete this file?

Reply:
Made a new commit that deletes the files with Chinese translations. Obviously another PR has to be made to the https://github.com/emqx/emqx-i18n repo later. If this can be merged before that repo is updated and if you think this PR is ok now, I would really appreciate if you could help me merge this as I would like to avoid conflicts (again).

Reply:
PR for adding changing the translation repository:

https://github.com/emqx/emqx-i18n/pull/3